gitbook-docs/put.md

```diff
@@ -1,11 +1,12 @@
 # PUT Requests
 
-PUT requests are sent to a server to request a change to a value. For example, a client would use PUT to switch the
-anchor light on or off, change the heading of the autopilot, or set position of the anchor.
+PUT requests are sent to a server to request an action to be taken. For example, a client would use PUT to switch the anchor light on or off, change the heading of the autopilot, or change the current input on a stereo.
+
+Note that this is very different than updating the current state of something. Use a delta update message to report on somethings current value. For example. to report on the current wind speed from a sensor. See [Delta Format](data_model.md)
```

Comment:
I would leave the first sentence out, because imho it suffers from the same confusion _update the current state_ versus _report the current value_.

_somethings current value_ => "the current value of someting"?

Punctuation is off in _For example._

I usually try to express everything so that the link is inline and not use a separate _See X_ expression.